API.md

```diff
@@ -879,6 +879,21 @@ openmct.install(openmct.plugins.CouchDB('http://localhost:9200'))
 * `openmct.plugins.Espresso` and `openmct.plugins.Snow` are two different
   themes (dark and light) available for Open MCT. Note that at least one
   of these themes must be installed for Open MCT to appear correctly.
+* `openmct.plugins.URLIndicatorPlugin` adds an indicator which shows the
+availability of a URL with the following options: 
+  - `url` : URL to indicate the status of
+  - `icon`: Icon to show in the status bar, defaults to `database`
```

Comment:
Two things here -
1. Let's just make this 'glyphClass' and remove the `icon` prefix in the code. That way users can use their own icon if they want to.
2. Let's also point users to the [style guide](https://nasa.github.io/openmct/style-guide/) so that they can see the icons that are available to them out of the box.

Reply:
@akhenry @ev1stensberg for number 1, use `cssClass` for consistency with the rest of the codebase.

Reply:
@larkin Ah, yep. Indicators specify several different css classes (`cssClass`, `glyphClass`, etc.), and I made the silly assumption that `glyphClass` was the one used to specify the glyph used, but it's not :)